stage2/04-pngview/files/splashscreen.service

```diff
@@ -1,13 +1,12 @@
 [Unit]
 Description=Splash screen
 DefaultDependencies=no
-After=local-fs.target
 
 [Service]
-ExecStart=/bin/bash -c "/usr/bin/pngview /opt/splash.png -b $(if [ $(cat /proc/device-tree/model | awk '{print $3}') == "4" ]; then echo 0x643f; else echo 0x346f; fi)"
+ExecStart=/usr/bin/show-splashscreen /usr/share/splash_images/ /home/pyxis/.inox/inox-config.yaml
```

Comment:
This does not look right that `/home/pyxis/.inox/inox-config.yaml` creeps into here.
What about this, we use a fixed location for the splash screen image, but that location is actually a symlink that is updated every time orientation setting is changed or screen resolution change is detected?